README.md

```diff
@@ -14,15 +14,41 @@
 
 ## Install
 
-Add a line to call the `init.sh` file from your boot `/etc/rc.local` on raspbian lite
+### What you need to run the project
+
+ - a Raspberry Pi with Wifi, or ethernet connection if you like cables
+ - an SD card (2GB should do...)
+ - 2 leds (red & green, or anything you see fit) with 330Ω resitors and cables
+ - a push button with cables
+ - optionnaly a breadboard, to check the connections before soldering the elements
+
+![LED](images/Red_LED.png "Red LED example")
+![Resistor](images/330OhmResistor.png "330Ω Resistor")
+
+### How to build
+
+ * Connect the push button on pins 5 & 6 _(SCL & Ground)_, so the click will wake it.
+ * Connect the green led and a resistor on pins 32 & 34 _(PWM0 & Ground)_.
+ * Connect the red led and the other resistor on pins 12 & 14 _(PWM0 & Ground)_.
+
+![GPIO connectors](images/Raspberry-GPIO-Pinout.png "GPIO connectors")
+
+### How to setup
+
+ * Get a Raspbian lite image from the [official page](https://www.raspberrypi.org/downloads/raspbian/)
+ * Burn it on the SD card: ex. on Linux/BSD -> `sudo dd bs=1M conv=noerror,sync status=progress if=<path_to_your_img/name.img> of=<path_to_mounted_SDcard>`
+ * Start the pi with screen and keyboard to configure Wifi, using the `sudo raspi-config` command
+ * Copy source code to `/home/pi/coffee-button` on the pi
+ * Copy `.env.example` to `.env` and add the webhook and token you created
```

Comment:
Préciser quels sont les tokens attendus et de quel site ils proviennent ?

Reply:
Je pensais mettre un exemple, plutôt. Pour laisser le choix sur la solution utilisée.
Ex: We choose Zapier to handle the request and filter the Authorization header then send the email and Slack notification.